app/src/main/java/com/beemdevelopment/aegis/ui/dialogs/Dialogs.java

```diff
@@ -515,6 +515,24 @@ public static void showImportersDialog(Context context, boolean isDirect, Import
                 .create());
     }
 
+    public static void showPartialGoogleAuthImportWarningDialog(Context context, List<Integer> missingIndexes, int entries, DialogInterface.OnClickListener dismissHandler) {
+        String missingIndexesAsString = missingIndexes.stream()
+                .map(index -> String.format("• Index %d", index + 1))
```

Comment:
I think we should come up with something more user friendly and translatable for "Index" here.